schemas/styles/csl.rnc

```diff
@@ -649,6 +650,72 @@ div {
 }
 # ==============================================================================
 
+## cs:title Rendering Element
+div {
+  ## Use to render titles if "title-main" and "title-sub" need different formatting
+  ## or to omit either "title-main" or "title-sub"
+  rendering-element.title =
+    element cs:title {
+      title.attributes,
+      title.formatting,
+      (title.main?, title.sub?)
```

Comment:
Have you tested this on a validating and tag-completing editor like Atom?

I think the way that's defined, this would be valid:

``` xml
<cs:title>
  <cs:main/>
  <cs:sub/>
</csL:title>
```

... but this would not:

``` xml
<cs:title>
  <cs:sub/>
  <cs:main/>
</csL:title>
```

Reply:
I have used `cs:names` as a template. That's why the order matters here. But I agree with you that this probably makes things overly complex. I wouldn't mind removing that constraint if there are no compelling reasons for keeping it.